app/controllers/admin/works_controller.rb

```diff
@@ -96,6 +96,33 @@ def submit_ohms_xml
     end
   end
 
+
+  # Create_combined_audio_derivatives in the background, if warranted.
+  # PATCH/PUT /admin/works/ab2323ac/create_combined_audio_derivatives
+  # Unfortunately, if the job fails for any reason, the user will not be notified.
+  def create_combined_audio_derivatives
+    unless @work.genre.include? 'Oral histories'
+      redirect_to admin_work_path(@work), flash: { error: "This isn't an oral history." }
+      return
+    end
+    unless work_audio_members.present?
+      redirect_to admin_work_path(@work, anchor: "nav-oral-histories"), flash: { error: "This oral history doesn't have any audio files." }
+      return
+    end
+
+    current = @work.oral_history_content!.combined_audio_fingerprint
+    if current != CombinedAudioDerivativeCreator.new(@work).fingerprint
+      # As things currently stand, the job can fail silently if e.g. the
+      # originals can't be downloaded from AWS for any reason.
+      CreateCombinedAudioDerivativesJob.perform_later(@work)
+      notice = "The combined audio derivative job has been launched."
+    else
+      notice = "We already have up-to-date combined audio for this item."
+    end
```

Comment:
I would not include the code to prevent running if the fingerprint is already up to date. Maybe it's corrupt or was done with the wrong settings and needs to be re-done -- there's no reason to forbid doing it again, and it's extra code, and the most bug-free code is the code you don't include. 

Not saying you need to change the UI at present, but let's take out the guard here that refuses to create if the fingerprint is up to date -- if the action is triggered, fire off the bg job.

Reply:
Yeah, that makes sense. Will remove.